.gitignore

```diff
@@ -23,3 +23,6 @@ erl_crash.dump
 # variables.
 /config/prod.secret.exs
 cover
+
+# allow npm to get the last version of tachyons see also postinstall in package.json
+	web/static/css/tachyons.min.css
```

Comment:
web/static/css/tachyons.min.css is already part of the repo, not sure if you want to keep the file?